ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html

```diff
@@ -178,16 +178,10 @@ <h2 class="social-media-header__text" translate>
                     <p class="big bold" translate>HomePage.SocialMediaSection.TwitterFeedTitle</p>
                     <app-twitter-timeline-widget></app-twitter-timeline-widget>
                 </div>
-                <div class="instagram-feed">
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
-                    <img class="image" src="https://via.placeholder.com/200x200" alt="Placeholder">
+                <div>
+                    <p class="big bold" translate>HomePage.SocialMediaSection.InstagramFeedTitle</p>
+                    <toolkit-instagram-feed-lightswitch-plugin [widgetId]="lightswitchWidgetId">
+                    </toolkit-instagram-feed-lightswitch-plugin>
                 </div>
```

Comment:
Only these changes are correct.

Reply:
@fat23cat Currently working with @zyme to straighten out this branch. Something went awry during the transition to new repo...